pkg/scorecard_result.go

```diff
@@ -25,6 +25,7 @@ import (
 	"strings"
 
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged